doc/src/stdlib/test.md

```diff
@@ -26,7 +26,7 @@ Base.Test.@test_throws
 
 For example, suppose we want to check our new function `foo(x)` works as expected:
 
-```julia
+```julia-repl
```

Comment:
In principle this should be doable, but it's a bit more work. I'd leave this for another day.